pkg/k8shandler/cluster.go

```diff
@@ -47,22 +48,22 @@ func CreateOrUpdateElasticsearchCluster(dpl *v1alpha1.Elasticsearch, configMapNa
 		return err
 	}
 	// TODO: get rid of this message as part of LOG-206
-	logrus.Infof("cluster %s required action is: %v", dpl.Name, action)
+	// logrus.Infof("cluster %s required action is: %v", dpl.Name, action)
```

Comment:
Is it worth keeping this and making it a `Debugf`?

Reply:
log-206 merged